src/java/io/bazel/rulesscala/test_discovery/DiscoveredTestSuite.scala

```diff
@@ -0,0 +1,45 @@
+package io.bazel.rulesscala.test_discovery
+
+import org.junit.runner.RunWith
+import org.junit.runner.Runner
+import org.junit.runners.Suite
+import org.junit.runners.model.RunnerBuilder
+
+@RunWith(classOf[TextFileSuite])
+class DiscoveredTestSuite
```

Comment:
I think documenting the architecture of how you are doing this is a good idea. To be honest, I'm only probably 80% aware how, and since I won't be using this code, in 6 months I'll probably be at 10% and may have to review patches at that time. I'd really appreciate a paragraph here explaining the approach rather than someone having to go read about junit architecture, then put together the approach from integrating the .bzl and scala code.